packages/braid-design-system/src/lib/components/MenuItem/useMenuItem.tsx

```diff
@@ -155,7 +155,11 @@ export function useMenuItem<MenuItemElement extends HTMLElement>({
           onClick();
         }
       },
-      background: isHighlighted ? hoverBackground : undefined,
+      /*
+      On mobile, using 'undefined' as the fallback background converts the element to a `ColoredBox` on touch.
+      This can interrupt href navigation and click events on iOS.
```

Comment:
```suggestion
      On mobile, switching between 'undefined' and a specified background converts the element to a `ColoredBox` on touch.
      This can interrupt href navigation and click events on iOS.
```